modules/Bio/EnsEMBL/G2P/Utils/Downloads.pm

```diff
@@ -219,9 +219,9 @@ sub write_data {
     my $confidence_category = ($confidence_category_attrib) ? $confidence_category_attribs->{$confidence_category_attrib} : 'No confidence category';
     my $allelic_requirement = ($ar_attrib) ? $allelic_requirement_attribs->{$ar_attrib} : undef;
     my $mutation_consequence = ($mc_attrib) ? $mutation_consequence_attribs->{$mc_attrib} : undef;
-    my $cross_cutting_modifier = ($ccm_attrib) ? join(',', map{$cross_cutting_modifier_attribs->{$_} } split(',', $ccm_attrib) ): undef;
+    my $cross_cutting_modifier = ($ccm_attrib) ? join(' ; ', map{$cross_cutting_modifier_attribs->{$_} } split(',', $ccm_attrib) ): undef;
     my $mutation_consequence_flag = ($mcf_attrib) ? $mutation_consequence_flag_attribs->{$mcf_attrib} : undef;
-    my $clinical_review_flag =  $clinical_review == 1 ? "Requires clinical review" : undef;
+    my $clinical_review_flag =  $clinical_review == 1 ? "Requires clinical review" : undef if defined($clinical_review);
```

Comment:
You could add the check to the existing condition:
my $clinical_review_flag =  ($clinical_review && $clinical_review == 1) ? "Requires clinical review" : undef;